From a2a2372eb588474073873cd745164e22ef076328 Mon Sep 17 00:00:00 2001
From: Mehdi Sabwat <mehdisabwat@gmail.com>
Date: Mon, 9 Sep 2019 19:07:49 +0200
Subject: [PATCH 11/15] configure: disable deprecated GL functions for
 emscripten

The build system assumes OpenGL functions are implemented if the headers are defined.
Which is wrong, so we need to disable the HAVE_GL marker.
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 5f24ab5945..66a69a91d7 100644
--- a/configure.ac
+++ b/configure.ac
@@ -3231,6 +3231,7 @@ PKG_CHECK_MODULES([GL], [gl], [
   ])
   AC_MSG_RESULT([${have_gl}])
 ])
+AS_IF([test "${SYS}" = "emscripten"], [have_gl="no"], [have_gl="yes"])
 AM_CONDITIONAL([HAVE_GL], [test "${have_gl}" = "yes"])
 AS_IF([test "${have_gl}" = "yes"], [
   AC_DEFINE([HAVE_GL], 1, [Defined if having OpenGL])
-- 
2.23.0

